### PR TITLE
Upgrade node to v16

### PR DESCRIPTION
**Pull request by ericboucher**
Upgrading node to v16. This solves `dyld` issues that may arise on recent Macbooks with M1 processors.

TODO
- [x] QA website
- [x] QA api

### CODE REVIEW

File: packages/website/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "cross-env EXTEND_ESLINT=true react-scripts start",
     "build": "cross-env EXTEND_ESLINT=true react-scripts build",
-    "test": "react-scripts --expose-gc test --logHeapUsage --maxWorkers=2",
+    "test": "react-scripts --expose-gc test --logHeapUsage --maxWorkers=2 --env=jsdom",
```
Comment (ericboucher) [inline packages/website/package.json:8]: is --env=jsdom still needed?

File: packages/website/package.json [hunk 2]
```diff
@@ -20,7 +20,10 @@
     "@material-ui/lab": "^4.0.0-alpha.56",
     "@material-ui/pickers": "^3.2.10",
     "@reduxjs/toolkit": "^1.3",
-
+    "@testing-library/dom": "^7.8.0",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.3.2",
+    "@testing-library/user-event": "^7.1.2",
```
Comment (ericboucher) [inline packages/website/package.json:26]: is all of this needed?

Comment (tleraj) [reply packages/website/package.json:26]: Just a line re-arrangement from `yarn`, no version changes, so I think it's fine :wink:

File: packages/website/package.json (hunk 1 above)
Comment (tleraj) [reply packages/website/package.json:8]: Let's find out :stuck_out_tongue:

Comment (tleraj) [reply packages/website/package.json]: https://github.com/aqualinkorg/aqualink-app/pull/650/commits/f7140d1117f108d4ca4f1ab88096c45536e0a111 suggests that this flag is not necessary.

**Review by tleraj — APPROVED** [review]
Looks good!! Once we QA, it should be good to go :slightly_smiling_face:

**Pull request merged**